src/main/java/teammates/storage/api/CommentsDb.java

```diff
@@ -388,9 +388,9 @@ public void deleteCommentsByInstructorEmail(String courseId, String email) {
 
         List<Comment> giverComments = this.getCommentEntitiesForGiver(courseId, email);
         // for now, this list is empty
-//        List<Comment> recipientComments = this.getCommentEntitiesForRecipients(courseId,
-//                CommentRecipientType.INSTRUCTOR, email);
-//        getPM().deletePersistentAll(recipientComments);
+        // List<Comment> recipientComments = this.getCommentEntitiesForRecipients(courseId,
```

Comment:
@xpdavid ok I see.
@whipermr5  lol I'm just feeling strange when I read a code commented this way. but since CheckStyle complains I think the update is fine.